### PR TITLE
Add ability to disable environment substitution in values files

### LINKED ISSUE #280: Ability to disable env substitution only in value files
Env variable substitution in desired state files is a very useful feature and I use it for dynamically generated test environments, but now when the substitution has been extended to the values files it causes problems with accidental value substitution (e.g. a lot of randomly generated passwords and command definitions contain `$` sign). I would be cool to disable substitution in value files so the state file could remain dynamic and values files static.

I can prepare PR if this functionality is acceptable.

### PR DESCRIPTION
**Pull request by medzin**
Add ability to disable environment substitution in values files.

Resolves #280

### CODE REVIEW

**Comment by mkubaczyk** [comment]
lgtm +1

File: utils.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func fromTOML(file string, s *state) (bool, string) {
 	}
 	addDefaultHelmRepos(s)
 	resolvePaths(file, s)
-	if !noEnvSubst {
+	if !noEnvSubst || !noEnvValuesSubst {
```
Comment (sami-alajrami) [inline utils.go]: I think this should be `&&` .. the current condition means that if `noEnvSubst = false` and `noEnvValuesSubst = true` , environment variables will still be substituted in values files.

File: utils.go [hunk 2]
```diff
@@ -107,7 +107,7 @@ func fromYAML(file string, s *state) (bool, string) {
 	addDefaultHelmRepos(s)
 	resolvePaths(file, s)
 
-	if !noEnvSubst {
+	if !noEnvSubst || !noEnvValuesSubst {
```
Comment (sami-alajrami) [inline utils.go]: same as above

File: utils.go (hunk 1 above)
Comment (medzin) [reply utils.go]: Yep, you are right - thanks.

**Comment by medzin** [comment]
@sami-alajrami fixed!

**Pull request merged**